environment.yml

```diff
@@ -6,36 +6,525 @@ channels:
-  - bioconda::freebayes=1.3.2
-  - bioconda::gatk4-spark=4.1.7.0
-  - bioconda::genesplicer=1.0
-  - bioconda::htslib=1.9
-  - bioconda::manta=1.6.0
-  - bioconda::msisensor=0.5
-  - bioconda::multiqc=1.8
-  - bioconda::qualimap=2.2.2d
-  - bioconda::samblaster=0.1.24
-  - bioconda::samtools=1.9
-  - bioconda::snpeff=4.3.1t
-  - bioconda::strelka=2.9.10
-  - bioconda::tiddit=2.7.1
-  - bioconda::trim-galore=0.6.5
-  - bioconda::vcfanno=0.3.2
-  - bioconda::vcftools=0.1.16
-  - conda-forge::pigz=2.3.4 
-  - conda-forge::r-ggplot2=3.3.0
+  - _libgcc_mutex=0.1=conda_forge
+  - _openmp_mutex=4.5=1_gnu
+  - _r-mutex=1.0.1=anacondar_1
+  - ascat=2.5.2=r40_2
+  - backports=1.0=py27_1
+  - backports.functools_lru_cache=1.6.1=py_0
+  - backports_abc=0.5=py27_0
+  - bcftools=1.9=ha228f0b_4
+  - binutils_impl_linux-64=2.35.1=h17ad2fc_1
+  - binutils_linux-64=2.35=hc3fd857_29
+  - bioconductor-biobase=2.50.0=r40h037d062_0
+  - bioconductor-biocgenerics=0.36.0=r40_0
+  - bioconductor-biocparallel=1.24.0=r40h5f743cb_0
+  - bioconductor-biostrings=2.58.0=r40h037d062_0
+  - bioconductor-delayedarray=0.16.0=r40h037d062_0
+  - bioconductor-dnacopy=1.64.0=r40h323e27b_0
+  - bioconductor-genomeinfodb=1.26.0=r40_0
+  - bioconductor-genomeinfodbdata=1.2.4=r40_0
+  - bioconductor-genomicalignments=1.26.0=r40h037d062_0
+  - bioconductor-genomicranges=1.42.0=r40h037d062_0
+  - bioconductor-iranges=2.24.0=r40h037d062_0
+  - bioconductor-matrixgenerics=1.2.0=r40_0
+  - bioconductor-noiseq=2.34.0=r40_0
+  - bioconductor-rhtslib=1.22.0=r40h037d062_0
+  - bioconductor-rsamtools=2.6.0=r40h5f743cb_0
+  - bioconductor-rtracklayer=1.50.0=r40h9bb0e53_1
+  - bioconductor-s4vectors=0.28.0=r40h037d062_0
+  - bioconductor-summarizedexperiment=1.20.0=r40_0
+  - bioconductor-xvector=0.30.0=r40h037d062_0
+  - bioconductor-zlibbioc=1.36.0=r40h037d062_0
+  - biopython=1.76=py27h516909a_0
+  - blas=1.0=mkl
+  - brotlipy=0.7.0=py27h516909a_1000
+  - bwa=0.7.17=hed695b0_7
+  - bwa-mem2=2.0=he513fc3_1
+  - bwidget=1.9.14=0
+  - bz2file=0.98=py_0
+  - bzip2=1.0.8=h7f98852_4
+  - c-ares=1.17.1=h36c2ea0_0
+  - cairo=1.16.0=hcf35c78_1003
+  - cancerit-allelecount=4.0.2=ha228f0b_1
+  - certifi=2020.6.20=pyhd3eb1b0_3
+  - cffi=1.14.0=py27hd463f26_0
+  - chardet=3.0.4=py27h8c360ce_1006
+  - click=7.1.2=pyh9f0ad1d_0
+  - clustalw=2.1=hc9558a2_5
+  - cmake=3.17.0=h28c56e5_0
+  - cnvkit=0.9.6=py27_1
+  - coloredlogs=14.0=py27h8c360ce_1
+  - colormath=3.0.0=py_2
+  - control-freec=11.6=he1b5a44_0
+  - cryptography=2.8=py27h2c19f6a_2
+  - curl=7.71.1=he644dc0_8
+  - cutadapt=1.18=py27h14c3975_1
+  - cycler=0.10.0=py27_0
+  - decorator=4.4.2=py_0
+  - ensembl-vep=99.2=pl526hecc5488_0
+  - enum34=1.1.10=py27h8c360ce_1
+  - fastqc=0.11.9=0
+  - fgbio=1.1.0=0
+  - font-ttf-dejavu-sans-mono=2.37=hab24e00_0
+  - font-ttf-inconsolata=2.001=hab24e00_0
+  - font-ttf-source-code-pro=2.030=hab24e00_0
+  - font-ttf-ubuntu=0.83=hab24e00_0
+  - fontconfig=2.13.1=hba837de_1004
+  - fonts-conda-ecosystem=1=0
```

Comment:
Thats quite a lot of fonts actually. For which bits are these required ...? The FastQC recipe was fixed at some point by Devon & Me, so shouldn't require any of these. Might be that other of your packages need it but I can't tell :-(